payments/migrations/0001_initial.py

```diff
@@ -0,0 +1,266 @@
+                        enum=payments.enums.ServiceType,
+                        max_length=40,
+                        verbose_name="service",
+                    ),
+                ),
+                (
+                    "period",
+                    enumfields.fields.EnumField(
+                        enum=payments.enums.PeriodType, max_length=8
+                    ),
+                ),
+            ],
+        ),
+        migrations.CreateModel(
+            name="BerthPriceGroup",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                (
+                    "name",
+                    models.CharField(
+                        max_length=128,
+                        verbose_name="berth price group name",
+                        unique=True,
+                    ),
+                ),
+            ],
+            options={"abstract": False},
+        ),
+        migrations.CreateModel(
+            name="WinterStorageProduct",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                (
+                    "created_at",
+                    models.DateTimeField(
+                        auto_now_add=True, verbose_name="time created"
+                    ),
+                ),
+                (
+                    "modified_at",
+                    models.DateTimeField(auto_now=True, verbose_name="time modified"),
+                ),
+                (
+                    "price_value",
+                    models.DecimalField(
+                        decimal_places=2,
+                        max_digits=10,
+                        validators=[
+                            django.core.validators.MinValueValidator(Decimal("0.01"))
+                        ],
+                        verbose_name="price",
+                    ),
+                ),
+                (
+                    "tax_percentage",
+                    models.DecimalField(
+                        choices=[(Decimal("24.0"), "24.0")],
+                        decimal_places=2,
+                        default=Decimal("24.0"),
+                        max_digits=5,
+                        verbose_name="tax percentage",
+                    ),
+                ),
+                (
+                    "price_unit",
+                    enumfields.fields.EnumField(
+                        default="amount", enum=payments.enums.PriceUnits, max_length=10
```

Comment:
hm, `django-enumfields` does not pass `choices` to this autogenerated migrations file?

Reply:
It's strange: it doesn't pass it, but it actually only allows the right choices. Maybe it's handled directly at the Django level and not DB?

Reply:
probably, yeah